Sources/SearchableCollectionViewController/SearchableCollectionController.swift

```diff
@@ -62,17 +59,7 @@ open class SearchableCollectionController: SweetCollectionController {
     }
 }
 
-extension SearchableCollectionController: UICollectionViewDataSource {
-    open func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        return UICollectionViewCell()
-    }
-
-    open func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        return 0
-    }
-}
-
-extension SearchableCollectionController: UICollectionViewDelegate {
+extension SearchableCollectionController: UIScrollViewDelegate {
     open func scrollViewDidScroll(_ scrollView: UIScrollView) {
```

Comment:
That would be a hell of a bug, if a delegate method got called from a scroll view we're not the delegate for. 👀 It's being called because the subclass is setting it as the delegate.